src/Microsoft.ML.Ensemble/Trainer/Regression/RegressionEnsembleTrainer.cs

```diff
@@ -43,6 +43,7 @@ public sealed class Arguments : ArgumentsBase
             [TGUI(Label = "Output combiner", Description = "Output combiner type")]
             public ISupportRegressionOutputCombinerFactory OutputCombiner = new MedianFactory();
 
+            // TODO: If we make this public again it should be an *estimator* of this type of predictor, rather than the (deprecated) ITrainer.
```

Comment:
(nit) what's the difference between `TODO` and `REVIEW` ?

Reply:
`REVIEW` was something favored by someone once upon a time, whereas `TODO` is a more standard thing. (Indeed VS has tooling around TODO.) I'm dabbling in starting to use todo instead?

Reply:
OK, it would be good to have a single way of doing this in the code base.... Maybe an issue to move all `REVIEW` comments to `TODO` if we are going down this route?

Reply:
... Not doing that as part of this PR. I'd rather just not shift to using todo as I had if you insist I do it everywhere if I do it once.

---
In reply to: [234257534](https://github.com/dotnet/machinelearning/pull/1626#discussion_r234257534) [](ancestors = 234257534)

Reply:
My suggestion above was to log an issue to make our code base consistent on 1 standard way of logging a "someone should do this" comment. I never insisted you do it everywhere in this PR.